superset-frontend/src/explore/exploreUtils/index.js

```diff
@@ -87,7 +87,7 @@ export function getURIDirectory(endpointType = 'base') {
   ) {
     return '/superset/explore_json/';
   }
-  return '/superset/explore/';
+  return '/explore/';
```

Comment:
With this change saving chart won't work until another PR implementing saving chart isn't merged

<img width="637" alt="image" src="https://user-images.githubusercontent.com/15073128/176905186-e4004074-cc22-4a4a-96aa-3ab95304ce71.png">

Reply:
You're right. I added a workaround to fix it and a `TODO` to remove the workaround with the next PR.

```
// TODO: This will be removed in the next PR that will change the logic to save a slice
url = url.replace('/explore', '/superset/explore');
```